src/relayer/Relayer.ts

```diff
@@ -373,6 +373,18 @@ export class Relayer {
         profitClient.captureUnprofitableFill(deposit, realizedLpFeePct, relayerFeePct, gasCost);
       }
     } else if (selfRelay) {
+      // Prefer exiting early here to avoid fast filling any deposits we send. This approach assumes that we always
+      // prefer someone else to fill the deposits. By doing this, we ensure repayment on the origin chain, resulting
+      // in no net change in capital allocation for our account and avoiding the need to pay an LP fee.
```

Comment:
I don't think this comment is correct - avoiding self-fills from lite chains will result in the relayer moving its balance from the lite chain to the destination chain.

```suggestion
      // prefer someone else to fill the deposits.
```

Reply:
Very fair point